### PR TITLE
Build fix for Linux

### PR DESCRIPTION
**Pull request by egekorkan**
As documented at https://github.com/eclipse-thingweb/website/issues/50, we have a build issue. I have fixed Hugo version to 0.79.0 at Netlify and this puts a band aid on the issue. It was surprising to find that we store Hugo binaries in GitHub, which we should not do. Anyways, this PR is updating the binaries.

### CODE REVIEW

File: .hugo_build.lock
Comment (danielpeintner) [inline .hugo_build.lock:1]: Do we need this file?

File: README.md [hunk 1]
```diff
@@ -11,3 +11,8 @@ You can see the content in different ways locally:
 - Start the Hugo server with drafts enabled: `hugo server -D`
 - Build Hugo website  in `/public`: `hugo`
 - Build Hugo website with drafts enabled in `/public`: `hugo -D`
+
+## Hugo Versions
+
+Until we find a solution, it is recommended to use version 0.79.0 of Hugo.
+For convenience, you can find the Linux binary under [binaries](./binaries/) folder.
```
Comment (danielpeintner) [inline README.md:18]: It is somewhat difficult to know which version the binary is.
Does it still make sense to add the tar.gz of if with the proper name? 

Note: I do not have a strong preference either...

File: .hugo_build.lock
Comment (egekorkan) [reply .hugo_build.lock:1]: Deleted, indeed not needed if it is empty :D

File: README.md (hunk 1 above)
Comment (egekorkan) [reply README.md:18]: I thought of this too but I think we should not keep both files for sure, adds unnecessary overhead. The version is documented in readme so it is fine to remove the tar ball.

**Pull request merged**